src/instruments/src/Common/simvars/helpers.ts

```diff
@@ -0,0 +1,32 @@
+import { SimVarType, SimVarUnitName } from './types';
+
+export const getSimVarKey = (varType: SimVarType, name: string, unit: string) => `${varType}_${name}_${normalizeSimVarUnitName(unit)}`;
+
+export const normalizeSimVarUnitName = (unit: SimVarUnitName): SimVarUnitName => {
+    switch (unit) {
+    case 'bool':
+    case 'Bool':
+    case 'boolean':
+    case 'Boolean':
+        return 'bool';
+    case 'number':
+    case 'Number':
+        return 'number';
+    case 'Degrees':
+    case 'degree':
+        return 'degree';
+    case 'Percent':
+    case 'percent':
+        return 'percent';
+    case 'Feet':
+    case 'feet':
+    case 'feets':
+    case 'Feets':
+        return 'feet';
+    case 'Knots':
+    case 'knots':
+        return 'knots';
+    default:
+        return unit;
+    }
+};
```

Comment:
```suggestion
export const normalizeSimVarUnitName = (unit: SimVarUnitName): SimVarUnitName => {
    return unit.toLowerCase();
};
```

Reply:
We should probably add better typing for the simvar units and fix them where applicable to avoid having to sanitize/normalize the units. Is there a complete list of all accepted simvar units somewhere? I know there's better typing in the "simvar.d.ts" file under root typings but these are currently not exported.

Reply:
https://docs.flightsimulator.com/html/index.htm#t=Programming_Tools%2FSimVars%2FSimulation_Variable_Units.htm
Not sure on completeness of that list.

Reply:
I'd also suggest adjusting the unit names within the project to use the correct unit name straight of the batch.

Reply:
I have added all supported simvar typings and cleaned up all usage of simvars to conform with the correct units.
As these typings are currently only exported from the common folder within instruments I have added some temporary workaround for the globals typing for SimVars to accept the new typings in the get/set functions.